client/app/bundles/App/Components/Clinical/ItemCollection/Index.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import ItemCollectionProps from 'terra-clinical-item-collection/docs/props-table/ItemCollection.md';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-clinical-item-collection/docs/README.md';
+import { version } from 'terra-clinical-item-collection/package.json';
+
+// Example Files
+import ItemCollectionExample from 'terra-clinical-site/lib/examples/item-collection/ItemCollectionExample';
+
+const ItemCollectionExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
```

Comment:
My guess is the thought behind it was one could link to that anchor but I personally don't see much use for it. This code is more or less copy/pasted from https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-site/src/examples/item-collection/Index.jsx
I'm fine removing it if we think thats something we should do.